# HG changeset patch
# User Francesco Potorti <Potorti@isti.cnr.it>
# Date 1217574674 -7200
#      Fri Aug 01 09:11:14 2008 +0200
# Branch release-3-0-x
# Node ID 642af2e62b1f0a30839b41c1f373528036fbccbe
# Parent  77793a0fad3dda0dadf3b82e2b384af45dd2d734
fix incorrect axis location properties

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,14 @@
+2008-08-01  Francesco Potorti <Potorti@isti.cnr.it>
+
+	* graphics.cc (axes::properties::properties,
+	axes::properties::set_defaults, axes::properties::factory_defaults):
+	Exchange xaxislocation and yaxislocation property values.
+
 2008-07-28  John W. Eaton  <jwe@octave.org>
 
 	* error.cc (Flasterror, Flasterr): Unwind-protect error_state.
 
 2008-07-24  John W. Eaton  <jwe@octave.org>
 
 	* dirfns.cc (Fmkdir): If directory already exists, return status =
 	true, but also set error message.
diff --git a/src/graphics.cc b/src/graphics.cc
--- a/src/graphics.cc
+++ b/src/graphics.cc
@@ -1135,18 +1135,18 @@ axes::properties::properties (const grap
     ycolor (color_values (0, 0, 0)),
     zcolor (color_values (0, 0, 0)),
     xscale (radio_values ("{linear}|log")),
     yscale (radio_values ("{linear}|log")),
     zscale (radio_values ("{linear}|log")),
     xdir ("normal"),
     ydir ("normal"),
     zdir ("normal"),
-    xaxislocation ("bottom"),
-    yaxislocation ("left"),
+    xaxislocation ("left"),
+    yaxislocation ("bottom"),
     linewidth (0.5),
     view (),
     visible ("on"),
     nextplot ("replace"),
     outerposition (),
     activepositionproperty (radio_values ("{outerposition}|position")),
     __colorbar__ (radio_values ("{none}|north|south|east|west|northoutside|southoutside|eastoutside|westoutside"))
 {
@@ -1436,18 +1436,18 @@ axes::properties::set_defaults (base_gra
   ycolor = color_property ("black");
   zcolor = color_property ("black");
   xscale = radio_property (radio_values ("{linear}|log"));
   yscale = radio_property (radio_values ("{linear}|log"));
   zscale = radio_property (radio_values ("{linear}|log"));
   xdir = "normal";
   ydir = "normal";
   zdir = "normal";
-  xaxislocation = "left";
-  yaxislocation = "bottom";
+  xaxislocation = "bottom";
+  yaxislocation = "left";
   linewidth = 0.5;
 
   Matrix tview (1, 2, 0.0);
   tview(1) = 90;
   view = tview;
 
   visible = "on";
   nextplot = "replace";
@@ -1810,18 +1810,18 @@ axes::properties::factory_defaults (void
   m["ycolor"] = color_property ("black");
   m["zcolor"] = color_property ("black");
   m["xscale"] = radio_property (radio_values ("{linear}|log"));
   m["yscale"] = radio_property (radio_values ("{linear}|log"));
   m["zscale"] = radio_property (radio_values ("{linear}|log"));
   m["xdir"] = "normal";
   m["ydir"] = "normal";
   m["zdir"] = "normal";
-  m["xaxislocation"] = "bottom";
-  m["yaxislocation"] = "left";
+  m["xaxislocation"] = "left";
+  m["yaxislocation"] = "bottom";
   m["linewidth"] = 0.5;
 
   Matrix tview (1, 2, 0.0);
   tview(1) = 90;
 
   m["view"] = tview;
 
   m["visible"] = "on";
